kustomize/prometheus/values.yaml

```diff
@@ -1154,6 +1154,67 @@ serverFiles:
             action: replace
             target_label: node
 
+      # Scrape config for service endpoints.
```

Comment:
can we separate scrape configs to a different file? I think it will be helpful as scraping and alerting rules will high touch and its easy to have in a separate file when possible ?